pkgs/tools/archivers/7zz/default.nix

```diff
@@ -17,8 +16,6 @@ stdenv.mkDerivation rec {
   # GCC: https://github.com/nidud/asmc/issues/8
```

Comment:
This seems to have been solved BTW, and version 21.07 can be compiled with either `asmc` (that seems to have a working Linux version now) or `uasm`.

Reply:
Looks good. Are you planning on adding ascm/uasm so we can use the optimized version?

Reply:
Not sure. If I am going to do it, probably in another PR.